daviaugustos/chat-app
# TL-05: The user should be able to send a photo taken from the camera or gallery.

# Title

[TL-05](https://trashlab.notion.site/Mobile-Take-Home-Assignment-10bdb96ac1e78033af18cdfc06a04fee): The user should be able to send a photo taken from the camera or gallery.

## Description

📸 Implemented camera feature for capturing images.
🖼️ Created gallery image selection functionality.

## Screenshots

<img width="395" alt="image" src="https://github.com/user-attachments/assets/60e0c57a-328d-4394-9b32-12d80df992ce">
<img width="395" alt="image" src="https://github.com/user-attachments/assets/15da2549-cfae-4db7-b96a-b288ec25a9f8">
<img width="395" alt="image" src="https://github.com/user-attachments/assets/d3eeadf2-2795-4892-a451-ab3a4ae9b27a">
<img width="395" alt="image" src="https://github.com/user-attachments/assets/9bae531b-e648-4fef-94ba-2410b33a89b5">


## Testing

- Open the app;
- Click on a contact to open the chat message history;
- Type a message and hit the send button;
- Check if the sent message is added to the chat history;
- Double tap a message to like it;
- Tap the media icon to capture a new photo;
- Send the captured photo;
- Tap the media icon to choose an existing image;
- Send the selected image;
- Verify both sent images appear in the chat history.
